SH4RE/SH4RE/Components/DropdownMenu.swift

```diff
@@ -36,7 +36,7 @@ struct DropdownStyle: DisclosureGroupStyle {
             }
             HStack {
                 configuration.label
-                    .foregroundColor(noSelection ? .gray : .black)
+                    .foregroundColor(noSelection ? .textfield : .black)
```

Comment:
this change is so the dropdown menu default colour matches that of the TextField swift component

Reply:
sounds good!